volatility3/framework/constants/__init__.py

```diff
@@ -65,7 +66,10 @@
 LOGLEVEL_VVVV = 6
 """Logging level for four levels of detail: -vvvvvv"""
 
-CACHE_PATH = os.path.join(os.path.expanduser("~"), ".cache", "volatility3")
+
+CACHE_PATH = os.path.join(
+    os.environ.get("XDG_CACHE_HOME") or os.path.expanduser("~/.cache"), "volatility3"
```

Comment:
Just want to verify this will work on windows?  Usually we don't include the slash, but use `os.path.join` to ensure it will work on all operating systems...

Reply:
Ah true, I'd forgotten about that.  Still think it's better to let python handle the slashes.  Thanks for the quick fix up!  5:)